build.ps1

```diff
@@ -91,9 +91,8 @@ Invoke-BuildStep 'Creating artifacts' {
     } `
     -ev +BuildErrors
 
-Invoke-BuildStep 'Signing the packages' { 
-        $ProjectPath = Join-Path $PSScriptRoot "build\sign.proj"
-        Build-Solution $Configuration $BuildNumber -MSBuildVersion "$msBuildVersion" $ProjectPath `
+Invoke-BuildStep 'Signing the packages' {
+        Sign-Packages -Configuration $Configuration -BuildNumber $BuildNumber -MSBuildVersion "$msBuildVersion" `
```

Comment:
`"$msBuildVersion"` can probably be just `$msBuildVersion`

Reply:
Done.